frontend/packages/pipelines-plugin/src/components/pipelineruns/detail-page-tabs/PipelineRunDetailsSection.tsx

```diff
@@ -15,14 +16,14 @@ const PipelineRunDetailsSection: React.FC<PipelineRunDetailsSectionProps> = ({ p
     <>
       <SectionHeading text={t('pipelines-plugin~PipelineRun details')} />
       <PipelineRunVisualization pipelineRun={pipelineRun} />
-      <div className="row">
-        <div className="col-sm-6">
+      <Grid hasGutter>
+        <GridItem sm={6}>
           <ResourceSummary resource={pipelineRun} />
-        </div>
-        <div className="col-sm-6 odc-pipeline-run-details__customDetails">
+        </GridItem>
+        <GridItem sm={6} className="odc-pipeline-run-details__customDetails">
           <PipelineRunCustomDetails pipelineRun={pipelineRun} />
-        </div>
-      </div>
+        </GridItem>
+      </Grid>
```

Comment:
<img width="1449" alt="Screenshot 2025-05-06 at 5 18 49 PM" src="https://github.com/user-attachments/assets/4522e479-7a5b-453b-b663-0213d0b0eefd" />


for comparison - 4.19.0-0.nightly-2025-05-05-205331
<img width="1447" alt="Screenshot 2025-05-06 at 5 19 31 PM" src="https://github.com/user-attachments/assets/b41ff124-fed7-4be9-9e71-7042351f4e3f" />

Reply:
Unable to produce:

![image](https://github.com/user-attachments/assets/59cc7c4a-dc42-4f34-a493-02ec045d728c)